test/integration/acceptance/query_permission_test_ast.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "module/shared_model/builders/common_objects/account_asset_builder.hpp"
+#include "module/shared_model/builders/protobuf/common_objects/proto_account_asset_builder.hpp"
+
+#include "query_permission_test_ast.hpp"
```

Comment:
This include should be the first one, according to our [c++ style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
done